src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -94,7 +97,7 @@
     <RuntimeIdentifier  Condition=" '$(RuntimeIdentifiers)' != '' And '$(RuntimeIdentifier)' != '' " />
     <GenerateApplicationManifest Condition=" '$(GenerateApplicationManifest)' == '' ">true</GenerateApplicationManifest>
     <!-- Default to Mono's AOT in Release mode -->
-    <RunAOTCompilation Condition=" '$(RunAOTCompilation)' == '' and '$(AotAssemblies)' == '' and '$(Configuration)' == 'Release' and '$(PublishAot)' != 'true' ">true</RunAOTCompilation>
+    <RunAOTCompilation Condition=" '$(RunAOTCompilation)' == '' and '$(AotAssemblies)' == '' and '$(Configuration)' == 'Release' and '$(_AndroidNativeAot)' != 'true' ">true</RunAOTCompilation>
```

Comment:
I realize that this is pre-existing logic, but we really shouldn't be using `'$(Configuration)' == 'Release'` (or `== 'Debug'`, for that matter).  Looking at the structure of `Microsoft.Android.Sdk.DefaultProperties.targets`, I'm not sure why we have *this*`Condition` setup this way either, given that the file already has a block:

```xml
<PropertyGroup  Condition=" '$(Configuration)' == 'Release' ">
  …
</PropertyGroup>
```

Do we need to check `$(Configuration)` at all here?